code/modules/crafting/smeltery.dm

```diff
@@ -0,0 +1,46 @@
+/*Crafting recipes for stuff made out of the blacksmtihing tools*/
+
+/datum/crafting_recipe/broadsword
+	name = "Smith Broadsword"
```

Comment:
I think, since it's a recipe, it's emulating doing the action. The fact that some say 'smithed,' then, is a bit weird...